docs/releases/3.0.txt

```diff
@@ -494,6 +494,12 @@ Miscellaneous
   :class:`~django.db.models.OneToOneField` ``'_id'`` attribute now unsets the
   corresponding field. Accessing the field afterwards will result in a query.
 
+* The compatibility imports of ``django.core.exceptions.EmptyResultSet`` in
+  ``django.db.models.query`` and ``django.db.models.sql.datastructures`` have
+  been removed. Other compatibility imports that were also removed are
+  ``django.forms.utils.pretty_name`` and ``django.forms.boundfield.BoundField``
+  from ``django.forms.forms``
```

Comment:
How about two shorter notes that are easier to parse if you are searching for a particular failing import?

```
* The compatibility imports of ``django.core.exceptions.EmptyResultSet`` in
  ``django.db.models.query`` and ``django.db.models.sql.datastructures`` have
  been removed.

* The compatibility imports of ``django.forms.utils.pretty_name`` and
  ``django.forms.boundfield.BoundField`` from ``django.forms.forms`` have been
  removed.
```

Reply:
(following the tense of existing notes: "have been" -> are)